graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/GraphQLJavaErrorInstrumentationTest.kt

```diff
@@ -157,12 +164,49 @@ class GraphQLJavaErrorInstrumentationTest {
         Assertions.assertThat(result.errors[1].extensions["errorDetail"]).isEqualTo("FIELD_NOT_FOUND")
     }
 
+    @Test
+    fun `Error contains errorDetail and errorType in the extensions for serialization error`() {
+        val graphQL: GraphQL = buildGraphQL(schema)
+        val result = graphQL.execute(
+            """
+            {
+                ip
+            }
+            """.trimIndent()
+        )
+
+        Assertions.assertThat(result.errors.size).isEqualTo(1)
+        Assertions.assertThat(result.errors[0].extensions.keys.containsAll(listOf("errorDetail", "errorType")))
```

Comment:
This line isn't asserting anything.

Reply:
Hi @kilink , I have added `isTrue` assertion in this test.  At the same time, I see that similar assertion was missing in other tests also, so I have added the assertion in them as well.